packages/runtime/test-runtime-utils/src/test/types/validateTestRuntimeUtilsPrevious.ts

```diff
@@ -76,7 +76,7 @@ use_current_ClassDeclaration_MockContainerRuntime(
 
 /*
 * Validate back compat by using current type in place of old type
-* If breaking change required, add in package.json under typeValidation.broken:
+* If breaking change required, add in package.json under typeValidation.broken.0.58.2002:
```

Comment:
this looks like you are using an old version of the build-tools. i would not expect this change